json-rpc/docs/client_implementation_guide.md

```diff
@@ -316,7 +338,8 @@ Distinguish above four types errors can help application developer to decide wha
 
 Other than general error handling, another type of error that client / application should pay attention to is server side stale response. This type problem happens when a Full Node is out of sync with the Libra network, or you connected to a sync delayed Full Node in a cluster of Full Nodes. To prevent these problems, we need:
 - Track server side data freshness, Libra JSON-RPC server will always respond `libra_ledger_version` and `libra_ledger_timestampusec` (see [Libra Extensions](./../json-rpc-spec.md#libra-extensions)) for clients to validate and track server side data freshness.
-- Use http connection pool to keep connection alive and reuse, so that it's likely we can hit the same Full Node server and less likely to get in-consistent data because of hitting different Full Node in a cluster.
+- Retry query / get methods calls when response is from a stale server.
+- Do not retry for submit transaction call, because the submitted transaction can be synced correctly even you submitted it to a stale server. You may receive a JSON-RPC error if submitted same transaction.
```

Comment:
- Do not retry for submit transaction call, because the submitted transaction can be synced correctly even you submitted it to a stale server. You may receive a JSON-RPC error if same transaction was submitted twice